tyr/servers/server.py

```diff
@@ -790,6 +800,25 @@ def route(self, wait=False):
                     self.log.error(str(e))
                     raise e
 
+    def ingress_rules(self):
```

Comment:
Is this just adding ingress security group rules? Is it something that could be done with #32?